en/docs/develop/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -232,9 +232,17 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
 
 6.  If you sign in to the Publisher, then access the Developer Portal application, click **Login** and verify that the same user is already authenticated in the Developer Portal.
 
+!!!info
+    When Just-In-Time Provisioning is enabled, the user details will be saved in the API Manager user store. User profile details will be update via federation following each login event. To preserve the user profile details without any changes you need to enable the `SystemRolesRetainedProvisionHandler`.
```

Comment:
The above sentence, ("To preserve the user profile details without any changes you need to enable the 'SystemRolesRetainedProvisionHandler'.") needs to be corrected as follows,
"To preserve previously assigned system roles (Application/, Workflow/) in the user profile, you need to enable the 'SystemRolesRetainedProvisionHandler'."